services/src/main/java/org/exoplatform/addons/gamification/service/mapper/DomainMapper.java

```diff
@@ -26,86 +25,86 @@
 import org.exoplatform.services.log.ExoLogger;
 import org.exoplatform.services.log.Log;
 
-
 public class DomainMapper {
 
-    private static SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd");
-    private static final Log LOG = ExoLogger.getLogger(DomainMapper.class);
+  private static SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd");
 
```

Comment:
use DateTimeFormatter in Utils : https://github.com/Meeds-io/gamification/blob/feature/rewards/services/src/main/java/org/exoplatform/addons/gamification/utils/Utils.java#L24